### ISSUE TITLE
[Accessibility] p-multiselect and p-select: Missing aria-hidden=true and toggle-all button must be disabled if there are not results

### ISSUE THREAD

**Issue by Eragaus** [primefaces/primeng#19411]
### Describe the bug

The p-select have missing aria-hidden="true" on clear and dropdown icon.

For the p-multiselect, to be accessible, when there are not visible options, the toggle-all button must be disabled.

Because of those two cases, the components have a lack of accessibility.

### Pull Request Link

https://github.com/primefaces/primeng/pull/19410

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-zqnce3pk-gckxfejs?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html

### Environment

I am working on an Angular application using the last version of PrimeNG 21. The issue is reproducible on both Chrome and Edge. The issue is not related to the environment.

### Angular version

21.1

### PrimeNG version

v21

### Steps to reproduce the behavior

p-select:

1. Create a p-select with [showClear]="true"
2. Use Console Developer and browse through the different icons: dropdown icon and clear icon.
3. The aria-hidden attributes are missing and are not equals to true.
You can also specify a clearIcon and dropdownIcon.

p-multiselect:
1. Create a p-select with [showToggleAll]="true"
2. Place an empty array in the options or search for something that does not match the list.
3. Click on the toggle-all button.
Normally, the button should be disabled.

### Expected behavior

For the p-select, the clear and dropdown icon must have aria-hidden=true even we specify a clear icon or dropdownIcon.

For the p-multiselect, to be accessible, when there are not visible options, the toggle-all button must be disabled.
